app/server/lastoppFil.server.ts

```diff
@@ -0,0 +1,48 @@
+import { Session } from '@remix-run/node';
+
+import { IFil, IPostFilResponse } from '~/typer/filopplastning';
+import { EMiljø } from '~/typer/miljø';
+import { EStatusKode } from '~/typer/response';
+
+import { postFilMedToken } from './authorization';
+
+const hentFilOpplastningURL = () => {
+  const sti = '/api/dokument';
+
+  switch (process.env.ENV) {
+    case EMiljø.LOKAL:
+      return 'http://localhost:8099' + sti;
+    case EMiljø.MOCK:
+    case EMiljø.PRODUKSJON:
+    default:
+      return 'https://nav-familie-endringsmelding-api.fly.dev' + sti;
+  }
+};
```

Comment:
Kan være egen PR, men kan vi skrive om utils-funksjonene slik at det er en funksjon som henter kun det som skal foran `sti` og så en egen hvor man legger på sti? Da kan man samle hentUrl for både dok og send endring litt

Reply:
Enig! Denne her kan man fikse på. 
Kort for dette; https://trello.com/c/nXPZxBJ7/175-fikse-hent-sti-for-send-fil-endepunkt